prac_06/programming_language.py

```diff
@@ -0,0 +1,25 @@
+"""CP1404 Practical - programming language class
+Estimated time: 30 minutes
+Actual time: 35 minutes """
+
+
+class ProgrammingLanguage:
```

Comment:
Again, I notice that LW has sometimes included a docstring for the class header, e.g. "Represent information about a programming language." or some such.